exporter/internal/queue/queue_test.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package queue
+
+import (
+	"context"
+)
+
+func consume[T any](q Queue[T], consumeFunc func(context.Context, T) error) bool {
```

Comment:
Is this used in tests only now? Will it be cleaner to update tests instead?

Reply:
I agree. This function is called 21 times in total, and I thought it's easier to update those separately.